server/routes/__tests__/health.spec.js

```diff
@@ -1,14 +1,14 @@
 const request = require('supertest');
-const express = require('express')
+const express = require('express');
 
 const { createHealthRouter } = require('../health');
 
 describe('GET healthchecks', () => {
   let config;
   let app;
-  const BUILD_NUMBER = 'foo-number'
-  const GIT_REF = 'foo-ref'
-  const GIT_DATE = 'foo-date'
+  const BUILD_NUMBER = 'foo-number';
```

Comment:
having to format this as a commit managed to get in that escaped the linting check